src/python/pants/backend/jvm/subsystems/scoverage_platform.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import logging
+import os
+from pants.build_graph.injectables_mixin import InjectablesMixin
+from pants.subsystem.subsystem import Subsystem
+from pants.java.jar.jar_dependency import JarDependency
+from pants.backend.jvm.targets.jar_library import JarLibrary
+from pants.build_graph.address import Address
+
+logger = logging.getLogger(__name__)
+
+SCOVERAGE = "scoverage"
+
+class ScoveragePlatform(InjectablesMixin, Subsystem):
+  """The scoverage platform."""
+
+  options_scope = 'scoverage'
+
+  @classmethod
+  def register_options(cls, register):
+    super().register_options(register)
+    register('--enable-scoverage',
+      default=False,
+      type=bool,
+      help='Specifies whether to generate scoverage reports for scala test targets.'
+           'Default value is False. If True,'
+           'implies --test-junit-coverage-processor=scoverage.')
+
+    register('--blacklist-file',
+      type=str,
+      help='Path to files containing targets not to be instrumented.')
+
+    register('--scoverage-target-path',
+      default='//:scoverage',
```

Comment:
It looks like this target is bootstrapped by default. If a user overrides this, the default //:scoverage target is still injected into the build graph.